hs628_tools/Outlier_debugged.py

```diff
@@ -0,0 +1,35 @@
+import numpy as np
+import csv
+import matplotlib.pyplot as plt
+
+##Class Outliers:
+"""
+    Concept:
+    Returns list of data(list array) which fall outside of two standard deviations under the normal distribution curve.
+    
+    Parameters:
+    mean, standard deviation(std)
+
+    Returns: 
+    outliers   
+"""
+def __init__(self, mean, std, outliers):
+     self.mean = mean
+     self.std = std
+     self.outliers = outliers
+
+with open("data.csv") as f:
```

Comment:
You don't necessarily  need to read your CSV file inside your class. you can read it in main program and pass it as a parameter to the outliers method.

Reply:
Thanks again, I modified it too for the final work. :)